mdal/mdal_data_model.hpp

```diff
@@ -279,6 +279,8 @@ namespace MDAL
 
       virtual void addVertices( size_t vertexCount, double *coordinates );
       virtual void addFaces( size_t faceCount, size_t driverMaxVerticesPerFace, int *faceSizes, int *vertexIndices );
+      virtual void addEdges( size_t edgeCount, int *startVertexIndices, int *endVertexIndices );
```

Comment:
wondering if we shouldn't add new capability in https://github.com/lutraconsulting/MDAL/blob/5e8223ed283c15815089bcf7e5558e4b5660ac88/mdal/frmts/mdal_driver.hpp#L15 that driver is able to modify/write mesh frame (add faces/edges/vertices) ..? so at least we can know how we stand with driver support

Reply:
I do agree with you - see #361 - but I don't think that should be fixed in this PR and/or only for edges (which would create inconsistency) I suggest a separate dedicated PR to update the capabilities across all entities.

Reply:
ok